scripts/install.sh

```diff
@@ -3,8 +3,15 @@
 set -e
 set -o pipefail
 
+RUSTUP_COMMAND="curl https://sh.rustup.rs -sSf | sh -s -- -y"
+if [ ! -z $CI ]; then
+    default_compiler=`cat rust-toolchain`
+    echo "Setting the default compiler to ${default_compiler}"
+    RUSTUP_COMMAND="$RUSTUP_COMMAND --default-toolchain ${default_compiler}"
```

Comment:
@zrzka do you think `default_compiler` should be uppercase as well ? I seen "local" variables being lowercase in scripts before - but happy to change to uppercase, if that's more consistent ?

Reply:
@cyplo shell rule is that upper cased are envs & builtins, rest is kind of flexible like upper / lower, but consistent across scripts.

Reply:
I see your comment above now - will change.